# 客户端请求时发送的一个数字
int32 request_num
---
# 服务器响应发送的数据
string task
float32 vector0
float32 vector1
float32 vector2
